client/app/src/main/java/com/example/criengine/Adapters/SearchBooksListAdapter.java

```diff
@@ -32,11 +32,14 @@
     private ArrayList<Book> items;
     private Context context;
     private DatabaseWrapper dbw;
+    private Boolean search;
```

Comment:
I would rename this to reflect it's a boolean and be more descriptive
```suggestion
    private Boolean isSearchFragment;
```

Reply:
this would also need to be updated everywhere else